.travis.yml

```diff
@@ -25,6 +25,7 @@ before_script:
 script:
   # - bundle exec rspec spec -fd
   - bundle exec rubocop app -R --format simple
+  - bundle exec rspec
```

Comment:
You can add the option `-fd` allow easier debug ;)

Reply:
Just added that option. Much easier to debug this way. Thanks 😃